examples/ts-react-chat/src/routes/index.tsx

```
@@ -25,7 +25,18 @@ import {
   useTranscription,
 } from '@tanstack/ai-react'
+import {
+  ByokProvider,
+  isPasskeyStorageSupported,
+  memoryStorage,
+  passkeyStorage,
+  useByok,
+  withByok,
+} from '@tanstack/ai-byok/react'
 import { clientTools } from '@tanstack/ai-client'
+import type { ProviderId } from '@tanstack/ai-byok/react'
```

ESLint

[error] `@tanstack/ai-byok/react` type import should occur after import of `@tanstack/ai-react-ui`

(import/order)